### PR TITLE
feat: add hardcoded value display from BE

### PR DESCRIPTION
**Pull request by mbednarik**
Also misc BE fixes

### CODE REVIEW

File: frontend/src/components/HoldingDiffs/HoldingDiffTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			<Table.Td>
+				<Flex gap="sm" align="center">
+					{sharesChangePercentage}%{" "}
+					{getChangeIcon(sharesChangePercentage)}
+				</Flex>
+			</Table.Td>
+			<Table.Td>
+				<Flex gap="sm" align="center">
+					{weightChange} {getChangeIcon(weightChange)}
+				</Flex>
```
Comment (mbednarik) [inline frontend/src/components/HoldingDiffs/HoldingDiffTableRow.tsx:64]: These computations are ugly and probably incorrect, we'll need to find a way in which we want to calculate this

File: FundParser/FundParser/Controllers/CsvController.cs [hunk 2]
```diff
@@ -8,17 +9,22 @@ namespace FundParser.Controllers;
...
 
     [HttpPost(Name = "csv")]
     public async Task<IActionResult> CsvUpload()
     {
         // temporary endpoint to update holdings for testing purposes
-        await _fundCsvService.UpdateHoldings();
+        // await _fundCsvService.UpdateHoldings();
+        await _holdingDiffService
+            .CalculateAndStoreHoldingDiffs(new DateTime(2024, 3, 15),
+                new DateTime(2024, 3, 15));
```
Comment (mbednarik) [inline FundParser/FundParser/Controllers/CsvController.cs:27]: This needs to be adjusted into separate endpoints and stuff, I just commented it out here for now...

File: FundParser/FundParser/Program.cs [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
 builder.Services.AddControllers();
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy(name: customAllowSpecificOrigins,
+        policy  =>
+        {
+            policy.WithOrigins("http://localhost:5173")
```
Comment (mbednarik) [inline FundParser/FundParser/Program.cs:15]: Hardcoded 🤮

File: FundParser/FundParser/Controllers/CsvController.cs (hunk 2 above)
Comment (adlermichal) [reply FundParser/FundParser/Controllers/CsvController.cs:27]: https://github.com/mbednarik/pv260_project/issues/15

**Review by samuelgocnik — APPROVED** [review]
🏎️

**Pull request merged**